src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -470,13 +470,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 					new Sound(modElement.getWorkspace(),
 							getRandomItem(random, ElementUtil.getAllSounds(modElement.getWorkspace())));
 			fluid.rarity = getRandomString(random, Arrays.asList("COMMON", "UNCOMMON", "RARE", "EPIC"));
-			fluid.specialInfo = new ArrayList<>();
-			if (!emptyLists) {
-				fluid.specialInfo = StringUtils.splitCommaSeparatedStringListWithEscapes(
-						"info 1, info 2, test \\, is this, another one");
-			} else {
-				fluid.specialInfo = new ArrayList<>();
-			}
+			fluid.specialInformation = new StringProcedure(emptyLists ? null : "string1",

```

Comment:
I would prefer something like

`emptyLists  ? null : new StringProcedure(_true ? null : "string1", "info 1, info 2, test \\, is this, another one");`

so cases where neither is specified are also covered

Reply:
Done!